core/src/test/scala/akka/persistence/jdbc/util/BaseQueryTest.scala

```diff
@@ -0,0 +1,27 @@
+package akka.persistence.jdbc.util
+
+import akka.persistence.jdbc.SingleActorSystemPerTestSpec
+import slick.lifted.RunnableCompiled
+
+class BaseQueryTest extends SingleActorSystemPerTestSpec {
+  import profile.api._
```

Comment:
Please replace this with `import ExtendedPostgresProfile.api._` and get rid of `profile` property from the parent class.

Reply:
done